test/e2e-multicluster-test/testdata/app/example-envbinding-app.yaml

```diff
@@ -17,10 +17,7 @@ spec:
     - name: data-worker
       type: worker
       properties:
-        image: busybox
-        cmd:
-          - sleep
-          - '1000000'
```

Comment:
why change this image

Reply:
`crccheck/hello-world` can be very small. I fix some flaky test by making test image smaller.